Remotion/ObjectBinding/Web.Test/TestForm.aspx

```diff
@@ -14,49 +14,60 @@
 --%>
-
 <%@ Page Language="c#" CodeBehind="TestForm.aspx.cs" AutoEventWireup="false" Inherits="OBWTest.TestForm" %>
 
 <!DOCTYPE html PUBLIC "-//W3C//DTD XHTML 1.0 Transitional//EN" "http://www.w3.org/TR/xhtml1/DTD/xhtml1-transitional.dtd">
 <html xmlns="http://www.w3.org/1999/xhtml">
 <head>
   <title>Test Form</title>
   <remotion:HtmlHeadContents ID="HtmlHeadContents" runat="server" />
-  <script type="text/javascript">
+</head>
+<body>
+  <form id="form1" runat="server">
+  <asp:ScriptManager ID="ScriptManager" runat="server" EnablePartialRendering="true"
+    AsyncPostBackTimeout="3600" />
+  <input id="StringField" type="text" />
+  <input id="IntField" type="text" />
+  <input type="button" value="ASP.NET Ajax" onclick="DoAspNetAjaxCall()" />
+  <input type="button" value="jQuery Ajax" onclick="DoJQueryAjaxCall()" />
+  <div id="Result">
+  </div>
+  </form>
+<script type="text/javascript">
     function DoAspNetAjaxCall()
     {
-      $("#Result").text("");
+      document.getElementById("Result").textContent = "";
 
-      var intValueAsString = $("#IntField").val();
+      var intValueAsString = document.getElementById("IntField").value;
       intValueAsString = intValueAsString == '' ? 0 : intValueAsString;
       var params = {
-        stringValue: $("#StringField").val(),
+        stringValue: document.getElementById("StringField").value,
         intValue: parseInt(intValueAsString)
       };
       executingRequest = Sys.Net.WebServiceProxy.invoke("TestService.asmx", "DoStuff", false, params,
                                           function (result, context, methodName)
                                           {
                                             executingRequest = null;
-                                            $("#Result").text(result);
+                                            document.getElementById("Result").textContent = result;
                                           },
                                           function (err, context, methodName)
                                           {
                                             executingRequest = null;
-                                            $("#Result").text(err.get_message());
+                                            document.getElementById("Result").textContent = err.get_message();
                                           });
 
     }
     function DoJQueryAjaxCall()
     {
-      $("#Result").text("");
+      document.getElementById("Result").textContent = "";
 
-      var intValueAsString = $("#IntField").val();
+      var intValueAsString = document.getElementById("IntField").value;
       intValueAsString = intValueAsString == '' ? 0 : intValueAsString;
       var params = {
-        stringValue: $("#StringField").val(),
+        stringValue: document.getElementById("#StringField").value,
         intValue: parseInt(intValueAsString)
       };
-      $.ajax({
+      $.ajax({ // TODO keep this?
```

Comment:
okay, the entire test is set to up test the jQuery and ASP.NET XmlHttpRequest APIs. Since we're keeping a jquery dependency in the code base, we can keep invoking jquery.ajax here, just then switch it to the noconflict variant. We do drop all the other jquery stuff, besides the ajax call.

In theory, the entire test may become redundant with dropping jquery, but since it doesn't hurt us on the test page, I want to keep it in case we need to do some testing in the future.